app/views/books/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>My Books</h1>
+<ul>
+  <% @books.sort_by(&:title).each do|book| %>
+    <li>
+      <a href="/books/<%= book.id %>">
+        <%= book.title %>
+      </a>
```

Comment:
Rails has a `link_to` component that is more declarative: https://guides.rubyonrails.org/routing.html#generating-paths-and-urls-from-code

Reply:
Changed it now 👍